src/ui/components/AppWrapper/AppWrapper.tsx

```diff
@@ -136,7 +135,13 @@ const peerConnectRequestSignChangeHandler = async (
   event: PeerConnectSigningEvent,
   dispatch: ReturnType<typeof useAppDispatch>
 ) => {
-  //TODO: Handle logic for the accept/decline sing request
+  dispatch(
+    setQueueIncomingRequest({
+      id: "peer-connect-signing",
```

Comment:
quick q, does this ID need to be unique or it's OK to be like this if we get 2? (if it is, then we can leave it)

Reply:
Yes, it's ok